docs/index.md

```diff
@@ -241,7 +238,43 @@ dub run dls:bootstrap
 }
 ```
 
-> Note: The older [natebosch/dart_language_server](https://github.com/natebosch/dart_language_server) is now deprecated.
+> **Note**: The older [natebosch/dart_language_server](https://github.com/natebosch/dart_language_server) is now deprecated.
+
+### Dockerfile<a name="dockerfile"></a>
+
+1. Install the [Dockerfile Syntax Highlighting](https://packagecontrol.io/packages/Dockerfile%20Syntax%20Highlighting) package from Package Control for syntax highlighting.
+2. Install the [Dockerfile Language Server](https://github.com/rcjsuen/dockerfile-language-server-nodejs):
+
+        npm install -g dockerfile-language-server-nodejs
+
+3. Add to LSP settings' clients:
+
+```js
+"docker-langserver": {
+  "command": ["docker-langserver", "--stdio"],
+  "enabled": true,
+  "languageId": "dockerfile",
+  "scopes": ["source.dockerfile"],
+  "settings": {
+    "docker": {
+      "languageserver": {
+        "diagnostics": {
+          // string values must be equal to "ignore", "warning", or "error"
+          "deprecatedMaintainer": "warning",
```

Comment:
There are some mixed tabs and spaces in configuration object.

Reply:
Are there? I cannot see any tab here

![Unbenannt](https://user-images.githubusercontent.com/6579999/80225759-cab8cf80-864b-11ea-9e1b-18172c8f6812.png)

The code blocks use 2 spaces indentation per level, like the LSP settings file does, and so that the code blocks don't become too wide at the readthedocs page.
Is this maybe a git config issue?

Reply:
hmm, sorry then. I've copied this object from github-rendered view so and pasted into sublime-settings file. Maybe something went wrong along the way.